pom.xml

```diff
@@ -148,12 +150,6 @@
       <version>1.58</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci</groupId>
-      <artifactId>annotation-indexer</artifactId>
```

Comment:
this fails with class not found exceptions in producing javadoc after incrementalifying,
I can't see any reason for it being here, possibly historical